cardano-node/test/Test/Cardano/Node/FilePermissions.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 {-# OPTIONS_GHC -Wno-unused-imports #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Should be with the other Language pragmas

Reply:
Fixed!